_data/glossary/persistent-volume.yaml

```diff
@@ -0,0 +1,14 @@
+id: persistent-volume
+name: Persistent Volume
+full-link: /docs/concepts/storage/persistent-volumes/
+related:
+- statefulset
+- deployment
+- pod
+tags:
+- core-object
+- storage
+short-description: >
+  Persistent Volume is a kind of interface through two resource types, i.e. pv and pvc, and provides pod persistent storage service.
+long-description: |
+  Persistent Volume subsystem provides an API for users and administrators that abstracts details of how storage is provided from how it is consumed.
```

Comment:
This is a useful distinction, and would be a better place to refer to PVCs.

Reply:
suggestion:

> PersistentVolumes (PVs) provide an API that abstracts details of how storage is *provided* from how it is *consumed*. PVs are used directly in scenarios where storage can be be created ahead of time (static provisioning). For scenarios that require on-demand storage (dynamic provisioning), PersistentVolumeClaims (PVCs) are used instead.